tests/app/views/test_users.py

```diff
@@ -192,15 +192,16 @@ def test_can_post_a_buyer_user(self):
             '/users',
             data=json.dumps({
                 'users': {
-                    'emailAddress': 'joeblogs@email.com',
+                    'emailAddress': 'joeblogs@email.gov.uk',
+                    'phoneNumber': '01234 567890',
                     'password': '1234567890',
-                    'role': 'admin',
+                    'role': 'buyer',
                     'name': 'joe bloggs'}}),
             content_type='application/json')
 
         assert_equal(response.status_code, 201)
         data = json.loads(response.get_data())["users"]
-        assert_equal(data["emailAddress"], "joeblogs@email.com")
+        assert_equal(data["emailAddress"], "joeblogs@email.gov.uk")
```

Comment:
Could also add an additional `assert` here to test the phone number is stored OK.